solutions/search/get-started/semantic-search.md

```diff
@@ -0,0 +1,211 @@
+  
+To learn about role-based access control, check out [](/deploy-manage/users-roles/cluster-or-deployment-auth/user-roles.md).
+
+<!--
+TBD: What is the impact of this "optimized for vectors" option?
+-->
+
+## Create a vector database
+
+When you create vectors (or _vectorize_ your data), you convert complex and nuanced documents into multidimensional numerical representations.
+You can choose from many different vector embedding models. Some are extremely hardware efficient and can be run with less computational power. Others have a greater understanding of the context, can answer questions, and lead a threaded conversation.
+The examples in this guide use the default Learned Sparse Encoder ([ELSER](/explore-analyze/machine-learning/nlp/ml-nlp-elser.md)) model, which provides great relevance across domains without the need for additional fine tuning.
+
+The way that you store vectors has a significant impact on the performance and accuracy of search results.
+They must be stored in specialized data structures designed to ensure efficient similarity search and speedy vector distance calculations.
+This guide uses the [semantic text field type](elasticsearch://reference/elasticsearch/mapping-reference/semantic-text.md), which provides sensible defaults and automation.
+
+:::::{stepper}
+::::{step} Create an index
+An index is a collection of documents uniquely identified by a name or an alias.
+You can follow the guided index workflow:
+
+- If you're using {{es-serverless}}, go to **{{es}} > Home**, select the semantic search workflow, and click **Create a semantic optimized index**.
+- If you're running {{es}} locally, go to **{{es}} > Home** and click **Create API index**. Select the semantic search workflow.
+
+When you complete the workflow, you will have sample data and can skip to the steps related to exploring and searching it.
+Alternatively, run the following API request in the [Console](/explore-analyze/query-filter/tools/console.md):
+
+```console
+PUT /semantic-index
+```
+
+For an introduction to the concept of indices, check out [](/manage-data/data-store/index-basics.md).
+::::
+::::{step} Create a semantic_text field mapping
+Each index has mappings that define how data is stored and indexed, like a schema in a relational database.
+The following example creates a mapping for a single field ("content"):
+
+```console
+PUT /semantic-index/_mapping
+{
+  "properties": {
+    "content": {
+      "type": "semantic_text"
+    }
+  }
+}
+```
+
+When you use `semantic_text` fields, the type of vector is determined by the vector embedding model.
+In this case, the default ELSER model will be used to create sparse vectors.
+
+For a deeper dive, check out [Mapping embeddings to Elasticsearch field types: semantic_text, dense_vector, sparse_vector](https://www.elastic.co/search-labs/blog/mapping-embeddings-to-elasticsearch-field-types).
```

Comment:
Sidebar: this is classic case of information that should be in the docs, not sitting in a blog 

We should consider adding this guidance into https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/1922/solutions/search/vector